README.md

```diff
@@ -120,6 +117,45 @@ curl localhost:8080
 # Output: Hello World from PHP HTTP function!
 ```
 
+# Run your function in Cloud Run
+
+First you must have the [gcloud SDK][gcloud] installed and [authenticated][gcloud-auth].
+
+Additionally, you need to have a Google Cloud project ID for the
+[Google Cloud Project][gcp-project] you want to use.
+
+Next, build the container using the example `Dockerfile`:
+
+```sh
+docker build . \
+    -f vendor/google/cloud-functions-framework/examples/hello/Dockerfile \
+    -t gcr.io/$GCLOUD_PROJECT/my-cloud-function
+```
+
+> **NOTE**: Be sure to replace `$GCLOUD_PROJECT` with your Google Cloud project ID.
+
+Next, use the `gcloud` command-line tool to deploy to Cloud Run:
+
+```sh
+gcloud run deploy my-cloud-function \
+    --image=gcr.io/$GCLOUD_PROJECT/php-ff-example \
+    --platform managed \
+    --set-env-vars "FUNCTION_TARGET=helloHttp" \
+    --allow-unauthenticated \
+    --region $YOUR_REGION
```

Comment:
Should also add a `--project=$GCLOUD_PROJECT` param

Reply:
Done!